modules/core/src/test/java/org/apache/ignite/internal/processors/cache/IgniteCacheAtomicConcurrentUnorderedUpdateAllTest.java

```diff
@@ -53,15 +54,16 @@
     private static final int CACHE_SIZE = 1_000;
 
     /** Parameters. */
-    @Parameterized.Parameters(name = "cacheMode={0}, writeThrough={1}")
+    @Parameterized.Parameters(name = "cacheMode={0}, writeThrough={1}, near={2}")
```

Comment:
What does this test do? How is it relevant to the ticket?

Reply:
There were no checks for deadlocks for invokeAll operation and near caches. I've added these checks since warnings are removed for all operations (including invokeAll) on atomic caches and all types of atomic caches (including near caches).